assets/js/wc-gateway-ppec-order-review.js

```diff
@@ -2,6 +2,10 @@
 	'use strict';
 
 	$( 'form.checkout' ).on( 'click', 'input[name="payment_method"]', function() {
+		if ( $( '#payment' ).find( '.wc-gateway-ppec-cancel' ).length ) {
+			return;
```

Comment:
Please add a comment explaining why we are doing this (i.e. https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/commit/0a52e16275285425ca178237518dbfab7338fc07 ) so that maintainers don't have to run blame here